lib/scrabble.rb

```diff
@@ -1,7 +1,29 @@
 class Scrabble
 
-  def score(word)
-    1
+  attr_reader :word, :scoring
+
+  def initialize(word)
+    @word = word
+    @scoring = []
+  end
+
+
+  def score
+    #split all the words into characters...=array
+    #for each in array, access the value
+    #add the values and return the score.
+    if @word.length == 1
+      @scoring = point_values[@word.upcase]
+    elsif @word == nil
+      @scoring = 0
+    else
+      words = []
+      words << @word.split("")
+      @scoring = @words.map do |letter|
+        point_values[letter.upcase]
+        @scoring.sum
```

Comment:
This would have worked if were down below the end on line 25. Right now you are summing the original @scoring array and it is getting saved by map. The return result of this map would likely be an array of numbers, but not the ones you're looking for.